nixos/modules/services/networking/cgproxy.nix

```diff
@@ -0,0 +1,110 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cgproxy;
+  settingsFormat = pkgs.formats.json { };
+  settingsFile = settingsFormat.generate "config.json" cfg.settings;
+in
+{
+
+  options = {
+    services.cgproxy = {
+      enable = mkEnableOption (lib.mdDoc "cgproxy");
```

Comment:
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557